src/Form/CompoundBackendAdmin.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Drupal\islandora_solr\Form;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\islandora\Form\ModuleHandlerAdminForm;
+
+/**
+ * Compound backend form.
+ */
+class CompoundBackendAdmin extends ModuleHandlerAdminForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'islandora_solr_compound_backend_admin';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['islandora_solr.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('islandora_solr.settings');
+
+    $form['islandora_solr_compound_relationship_field'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Solr compound relationship field'),
+      '#description' => $this->t('Solr field containing the compound relationship. Defaults to RELS_EXT_isConstituentOf_uri_ms'),
+      '#default_value' => $config->get('islandora_solr_compound_relationship_field', 'RELS_EXT_isConstituentOf_uri_ms'),
```

Comment:
Should add `islandora_solr_compound_relationship_field` to `config/schema/islandora_solr.schema.yml` and the default value to `config/install/islandora_solr.settings.yml` and remove the second param, as it isn't used

Reply:
Rather than retyping this out, same applies to the other usages of `Config::get()` in this pull, both in the config form and when it gets used up above